### DISCUSSION TITLE
Navigate to BottomNavigation tab

### DISCUSSION THREAD

**Discussion by omaarse** [callstack/react-native-paper#2588] [Q&A]
Hello there, I've a question, how can i navigate from a StackNavigator to an specific tab on BottomNavigation?

Thank you in advance.

**Comment by raajnadar** [comment]
You can find the answer in the docs https://reactnavigation.org/docs/nesting-navigators/#navigating-to-a-screen-in-a-nested-navigator.

Hope that answers your question, if not let me know I will help you.

**Reply by omaarse** [reply]
Hello @raajnadar thank You for the reply, I tried but I didn't get it to work, this is what I have

const MainDrawerHome = () => (
    <Drawer.Navigator drawerContent={(props) => <HomeDrawerContent {...props} />}>
        <Drawer.Screen name="Home" component={HomeStackNavigator}/>
    </Drawer.Navigator>
)


const HomeStackNavigator = (props) => { 

....

    const [routes] = useState([
        { index: 0, key: 'route0' },
        { index: 1, key: 'route1' },
        { index: 2, key: 'route2' },
        { index: 3, key: 'route3' },
        { index: 4, key: 'route4' },
    ])

return <BottomNavigation
            shifting={false}
            navigationState={{ index, routes }}
            onIndexChange={setIndex}
            renderScene={renderScene} />

}

...

Then 

                               <RootStack.Screen
                                    name='HomeStack'
                                    component={MainDrawerHome}
                                />

                               <RootStack.Screen
                                    name='AnotherStack'
                                    component={AnotherStack}
                                />

So, what I want to do is, navigate from "AnotherStack" to a specific tab in BottomNavigation (HomeStackNavigator in this case).

Thank You.

**Reply by raajnadar** [reply]
Your example is not clear can you add a snack demo for me to check?

**Reply by omaarse** [reply]
I made this example: https://snack.expo.io/@gozhira/bottomnavigation thank You.

**Reply by omaarse** [reply]
@raajnadar

**Reply by omaarse** [reply]
I found a solution, I update the snack example with this solution.

Thank You.

**Reply by raajnadar** [reply]
Oh sorry I didn’t get time! Glad that you fixed it yourself.
